samples_project/Assets/SampleViewer/Samples/StreamLayer/StreamLayerWebSocketSubscribe.cs

```diff
@@ -19,7 +19,7 @@
 using Newtonsoft.Json.Linq;
 using Esri.HPFramework;
 
-[System.Serializable]
+[Serializable]
```

Comment:
Since `System` is already included, no need to include it here. (same for some below(